packages/smooth_app/lib/data_models/user_preferences.dart

```diff
@@ -57,4 +61,9 @@ class UserPreferences extends ChangeNotifier {
 
   String get themeColorTag =>
       _sharedPreferences.getString(_TAG_THEME_COLOR_TAG) ?? 'COLOR_TAG_BLUE';
+
+  Future<void> setUserCountry(final String countryName) async =>
+      _sharedPreferences.setString(_TAG_USER_COUNTRY, countryName);
+
+  String? get userCountry => _sharedPreferences.getString(_TAG_USER_COUNTRY);
```

Comment:
That's probably the iso2 code that we need to store, not the country name.

Reply:
Done